### PR TITLE
Bug 1937722: OCP 4.8: Passing oVirt certificate information to Terraform

### PR DESCRIPTION
**Pull request by janoszen**
This PR fixes BZ 1937722. Please see that BZ for details.

### CODE REVIEW

**Review by staebler — COMMENTED** [review]
Please improve on the commit message.
See https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format.
/approve

File: data/data/ovirt/variables-ovirt.tf [hunk 1]
```diff
@@ -19,6 +19,21 @@ variable "ovirt_password" {
   description = "The plain password of user to access Engine API"
 }
 
+variable "ovirt_cafile" {
```
Comment (Gal-Zaidman) [inline data/data/ovirt/variables-ovirt.tf:22]: I'm pushing to removing this variable and have it in the bundle so I think you can remove it

File: data/data/ovirt/main.tf [hunk 2]
```diff
@@ -1,7 +1,10 @@
 provider "ovirt" {
-  url      = var.ovirt_url
-  username = var.ovirt_username
-  password = var.ovirt_password
+  url       = var.ovirt_url
+  username  = var.ovirt_username
+  password  = var.ovirt_password
+  cafile    = var.ovirt_cafile
```
Comment (Gal-Zaidman) [inline data/data/ovirt/main.tf:5]: I'm pushing to removing this variable and have it in the bundle so I think you can remove it

**Pull request merged**